package.json

```
@@ -86,67 +86,67 @@
+    "@angular/common": "14.1.3",
+    "@angular/compiler": "14.1.3",
+    "@angular/core": "14.1.3",
+    "@angular/forms": "14.1.3",
+    "@angular/material": "14.1.3",
+    "@angular/platform-browser": "14.1.3",
+    "@angular/platform-browser-dynamic": "14.1.3",
+    "@angular/platform-server": "14.1.3",
+    "@angular/router": "14.1.3",
+    "@nrwl/angular": "14.5.10",
     "core-js": "^2.5.4",
     "eslint-etc": "^5.1.0",
     "opencollective": "^1.0.3",
-    "rxjs": "~7.4.0",
+    "rxjs": "7.5.6",
     "semver": "^7.3.6",
     "strip-json-comments": "3.1.1",
     "tslib": "^2.3.1",
     "zone.js": "0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "14.0.0-rc.1",
-    "@angular-eslint/builder": "13.2.0",
-    "@angular-eslint/eslint-plugin": "13.2.0",
-    "@angular-eslint/eslint-plugin-template": "13.2.0",
-    "@angular-eslint/schematics": "13.2.0",
-    "@angular-eslint/template-parser": "13.2.0",
-    "@angular/cli": "14.0.0-rc.1",
-    "@angular/compiler-cli": "14.0.0-rc.1",
-    "@angular/language-service": "~13.3.0",
+    "@angular-devkit/build-angular": "14.1.3",
+    "@angular-eslint/builder": "14.0.3",
+    "@angular-eslint/eslint-plugin": "14.0.3",
+    "@angular-eslint/eslint-plugin-template": "14.0.3",
+    "@angular-eslint/schematics": "14.0.3",
+    "@angular-eslint/template-parser": "14.0.3",
+    "@angular/cli": "~14.1.0",
+    "@angular/compiler-cli": "14.1.3",
+    "@angular/language-service": "14.1.3",
     "@babel/core": "7.9.0",
-    "@nrwl/cli": "14.1.5",
-    "@nrwl/cypress": "14.1.5",
-    "@nrwl/eslint-plugin-nx": "14.1.5",
-    "@nrwl/jest": "14.1.5",
-    "@nrwl/linter": "14.1.5",
-    "@nrwl/node": "14.1.5",
-    "@nrwl/nx-cloud": "14.0.3",
-    "@nrwl/tao": "14.1.5",
-    "@nrwl/workspace": "14.1.5",
+    "@nrwl/cli": "14.5.10",
+    "@nrwl/cypress": "14.5.10",
+    "@nrwl/eslint-plugin-nx": "14.5.10",
+    "@nrwl/jest": "14.5.10",
+    "@nrwl/linter": "14.5.10",
+    "@nrwl/node": "14.5.10",
+    "@nrwl/nx-cloud": "14.5.4",
+    "@nrwl/tao": "14.5.10",
+    "@nrwl/workspace": "14.5.10",
     "@octokit/rest": "^15.17.0",
-    "@schematics/angular": "^14.0.0-rc.1",
+    "@schematics/angular": "14.1.3",
     "@testing-library/cypress": "^7.0.1",
     "@types/fs-extra": "^2.1.0",
     "@types/glob": "^5.0.33",
     "@types/globby": "^8.0.0",
-    "@types/jasmine": "3.6.3",
+    "@types/jasmine": "4.0.3",
     "@types/jasminewd2": "^2.0.2",
     "@types/jest": "27.4.1",
     "@types/lodash": "^4.14.80",
     "@types/ncp": "^2.0.1",
-    "@types/node": "14.14.33",
+    "@types/node": "18.7.1",
     "@types/ora": "^1.3.1",
     "@types/prettier": "^1.19.0",
     "@types/rimraf": "^0.0.28",
     "@types/semver": "^7.3.9",
     "@types/shelljs": "^0.8.5",
-    "@typescript-eslint/eslint-plugin": "^5.18.0",
-    "@typescript-eslint/parser": "^5.18.0",
+    "@typescript-eslint/eslint-plugin": "5.35.1",
+    "@typescript-eslint/parser": "5.35.1",
+    "@typescript-eslint/utils": "^5.29.0",
```

Maybe safer to put the same version as `eslint-plugin` and `parser` ?

```suggestion
    "@typescript-eslint/utils": "5.35.1",
```

---

Good catch, thanks!